# test exit status: failure codes, nonzero 
import sys
print('starting')
sys.exit(42)

